srv/salt/_modules/osd.py

```diff
@@ -978,10 +978,13 @@ def takeover():
 
 
 def wait_until_available(osd_id):
-    """Wait slightly over a minute for an OSD to become available (use e.g. after restarting)"""
-    timeout=0.5
-    while timeout < 60:
-        _, stdout, _ = __salt__['helper.run']("ceph daemon osd.{} status 2>/dev/null | jq -r .state".format(osd_id))
+    """Wait for an OSD to become available (use e.g. after restarting)"""
+    timeout_limit = __pillar__.get('osd_available_timeout', 120)
```

Comment:
and I would prefer the default something power of two so it is close to the "effective".